_health-care/_js/_components/_hello-world.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { IndexLink, Link } from 'react-router'
+
+import DateInput from './date_input';
+
+class Nav extends React.Component {
+  render() {
+    return (
+      <div>
+        <Link to="/hello">Hello</Link><br/>
+        <Link to="/night">Night</Link><br/>
+        <IndexLink to="/">Home</IndexLink>
+      </div>
+    )
+  }
+}
+
+class Hello extends React.Component {
+  render() {
+    return(
+      <div>
+        <h2>Hello World</h2>
+
+        <div className="row">
+          <h3>Happy Date</h3>
+          <DateInput date={this.props.applicationData.happyDate}
```

Comment:
But there will be a problem on the state update. :-/  I was going to try to work through that today.

I know how to fix it using layers of functions...it just feels a little ugly.